packages/estransform/lib/sourcemaps.js

```diff
@@ -1,8 +1,6 @@
 import { readFile } from 'fs/promises';
 import path from 'path';
-import sourceMapDefault from '@parcel/source-map';
-
-const { default: SourceMapNode } = sourceMapDefault;
+import SourceMapNode from './parcel-source-map-default.cjs';
```

Comment:
this workaround is suggested in https://github.com/evanw/esbuild/issues/532
and it's similar to another one you use in `packages/rna-storybook/lib/csf-tools.cjs`